ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ethernet-interfaces.yml

```diff
@@ -36,6 +36,65 @@ ethernet_interfaces:
     switchport:
       port_security:
         enabled: true
+      enabled: false
+      trunk:
+        allowed_vlan: 10-11
```

Comment:
If we need to give only 10 here as allowed vlan then we have put it into quote. Can you take a look

Reply:
fixed